OpenRA.Mods.Common/Lint/CheckPlayers.cs

```diff
@@ -51,6 +51,14 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 				if (map.SpawnPoints.Value.Distinct().Count() != spawnCount)
 					emitError("Duplicate spawn point locations detected.");
 			}
+
+			foreach (var kv in map.ActorDefinitions)
+			{
+				var actorReference = new ActorReference(kv.Value.Value, kv.Value.ToDictionary());
+				var ownerName = actorReference.InitDict.Get<OwnerInit>().PlayerName;
+				if (!playerNames.Contains(ownerName))
+					emitWarning("Actor {0} is owned by unknown player {1}.".F(actorReference.Type, ownerName));
```

Comment:
Well, I went for error now as there is no excuse for this yaml problem and it crashes the editor (though that problably should get fixed). However `emitError` because Travis is not a reasonable argument. Just make it also exit with != 0 on warnings, maybe using a `--pedantic` flag or do it by default. The linter has almost no false positives.